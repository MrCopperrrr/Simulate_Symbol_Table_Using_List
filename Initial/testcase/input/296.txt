INSERT x number
INSERT y string
LOOKUP x
LOOKUP y
BEGIN
INSERT z string
LOOKUP z
BEGIN
INSERT m number
LOOKUP m
LOOKUP z
LOOKUP y
END
INSERT n string
LOOKUP n
LOOKUP x
END
INSERT o number
LOOKUP o
LOOKUP x