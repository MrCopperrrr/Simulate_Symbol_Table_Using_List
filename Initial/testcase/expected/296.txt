success
success
0
0
success
1
success
2
1
0
success
1
0
success
0
0